tapis-project/tapis-ui
# task/TUI-237 - fix file explorer breadcrumbs

## Overview:

File explorer breadcrumbs were incorrect

## Related Github Issues:

- [TUI-237](https://github.com/tapis-project/tapis-ui/issues/237)

## Summary of Changes:

- breadcrumbsFromPathname utility function no longer removes the last fragment on behalf of the caller
- breadcrumbsFromPathname checks for an empty string text link and does not return those as fragments. This guards against accidental trailing slashes generating empty links, or double slashes
- Files Layout breadcrumbs explicitly removes /files link on its own behalf
- FileExplorer links should now function

## Testing Steps:

1. Navigate to different parts of a directory structure in the main file browser to confirm File breadcrumbs work
2. Try navigating inside one of the FileExplorer based modals

## UI Photos:

## Notes:
